### DISCUSSION TITLE
macOS installation problem

### DISCUSSION THREAD

**Discussion by chicken666** [LizardByte/community#516] [Sunshine]
### Select Topic Area

Question

### Body

OS version:macOS Sequoia 15.0
After the command "brew install sunshine", sunshine doesn't install properly
logs are shown below
-------------------------
Mac-mini-2:~ heiji$ brew install sunshine
==> Downloading https://mirrors.tuna.tsinghua.edu.cn/homebrew-bottles/api/formul
#=#=#                                                                           
==> Downloading https://mirrors.tuna.tsinghua.edu.cn/homebrew-bottles/api/cask.j
#=#=#                                                                           
==> Fetching lizardbyte/homebrew/sunshine
==> Cloning https://github.com/LizardByte/Sunshine.git
Updating /Users/heiji/Library/Caches/Homebrew/sunshine--git
==> Checking out tag v0.23.1
HEAD is now at 8b21db64 Merge pull request #2388 from LizardByte/nightly
HEAD is now at 8b21db64 Merge pull request #2388 from LizardByte/nightly
Entering 'packaging/linux/flatpak/deps/org.flatpak.Builder.BaseApp'
Entering 'packaging/linux/flatpak/deps/shared-modules'
Entering 'third-party/Simple-Web-Server'
Entering 'third-party/TPCircularBuffer'
Entering 'third-party/ViGEmClient'
Entering 'third-party/build-deps'
Entering 'third-party/googletest'
Entering 'third-party/moonlight-common-c'
Synchronizing submodule url for 'enet'
Entering 'third-party/moonlight-common-c/enet'
Entering 'third-party/nanors'
Entering 'third-party/nv-codec-headers'
Entering 'third-party/nvapi-open-source-sdk'
Entering 'third-party/tray'
Entering 'third-party/wayland-protocols'
Entering 'third-party/wlr-protocols'
/Users/heiji/Library/Caches/Homebrew/sunshine--git/packaging/linux/flatpak/deps/org.flatpak.Builder.BaseApp
/Users/heiji/Library/Caches/Homebrew/sunshine--git/packaging/linux/flatpak/deps/shared-modules
/Users/heiji/Library/Caches/Homebrew/sunshine--git/third-party/Simple-Web-Server
/Users/heiji/Library/Caches/Homebrew/sunshine--git/third-party/TPCircularBuffer
/Users/heiji/Library/Caches/Homebrew/sunshine--git/third-party/ViGEmClient
/Users/heiji/Library/Caches/Homebrew/sunshine--git/third-party/build-deps
/Users/heiji/Library/Caches/Homebrew/sunshine--git/third-party/googletest
/Users/heiji/Library/Caches/Homebrew/sunshine--git/third-party/moonlight-common-c
/Users/heiji/Library/Caches/Homebrew/sunshine--git/third-party/moonlight-common-c/enet
/Users/heiji/Library/Caches/Homebrew/sunshine--git/third-party/nanors
/Users/heiji/Library/Caches/Homebrew/sunshine--git/third-party/nv-codec-headers
/Users/heiji/Library/Caches/Homebrew/sunshine--git/third-party/nvapi-open-source-sdk
/Users/heiji/Library/Caches/Homebrew/sunshine--git/third-party/tray
/Users/heiji/Library/Caches/Homebrew/sunshine--git/third-party/wayland-protocols
/Users/heiji/Library/Caches/Homebrew/sunshine--git/third-party/wlr-protocols
==> Installing sunshine from lizardbyte/homebrew
==> cmake -S . -B build -DBUILD_WERROR=ON -DOPENSSL_ROOT_DIR=/opt/homebrew/opt/o
==> make -j
Last 15 lines from /Users/heiji/Library/Logs/Homebrew/sunshine/02.make:
../../../../build/assets/web/assets/Navbar-48ec9d0d.css                       0.07 kB │ gzip:   0.09 kB
../../../../build/assets/web/assets/_plugin-vue_export-helper-cff45802.css  327.59 kB │ gzip:  54.06 kB
../../../../build/assets/web/assets/password-41ebda5b.js                      0.63 kB │ gzip:   0.41 kB
../../../../build/assets/web/assets/welcome-6454da3a.js                       0.66 kB │ gzip:   0.42 kB
../../../../build/assets/web/assets/pin-677ef343.js                           0.81 kB │ gzip:   0.43 kB
../../../../build/assets/web/assets/troubleshooting-76080e6f.js               1.36 kB │ gzip:   0.59 kB
../../../../build/assets/web/assets/ResourceCard-ea4a7cba.js                  1.67 kB │ gzip:   0.64 kB
../../../../build/assets/web/assets/index-1d511c0f.js                         2.06 kB │ gzip:   0.90 kB
../../../../build/assets/web/assets/Navbar-dbaf0800.js                        2.16 kB │ gzip:   0.94 kB
../../../../build/assets/web/assets/config-f4fb6dcb.js                        4.97 kB │ gzip:   2.10 kB
../../../../build/assets/web/assets/apps-966a1e70.js                         85.57 kB │ gzip:  25.93 kB
../../../../build/assets/web/assets/_plugin-vue_export-helper-fe085d29.js   327.84 kB │ gzip: 111.98 kB
✓ built in 1.16s
[ 54%] Built target web-ui
make: *** [all] Error 2

If reporting this issue please do so at (not Homebrew/brew or Homebrew/homebrew-core):
  https://github.com/lizardbyte/homebrew-homebrew/issues
---------------------
I've tried to reinstall brew and changed to another network connection, but the problem still persists.

**Comment by ReenigneArcher** [comment]
https://github.com/LizardByte/Sunshine/issues/3180

[a comment by an automated account is not shown]
